contact.html

```diff
@@ -0,0 +1,129 @@
+  <link rel="stylesheet" type="text/css" href="resources/css/style.css">
+  <link rel="stylesheet" type="text/css" href="resources/css/responsive.css">
+  <title>Rmedia</title>
+
+  <!-- Favicons -->
+  <link rel="apple-touch-icon" sizes="180x180" href="/resources/favicons/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/resources/favicons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/resources/favicons/favicon-16x16.png">
+  <link rel="manifest" href="/resources/favicons/site.webmanifest">
+  <link rel="mask-icon" href="/resources/favicons/safari-pinned-tab.svg" color="#5bbad5">
+  <link rel="shortcut icon" href="/resources/favicons/favicon.ico">
+  <meta name="msapplication-TileColor" content="#da532c">
+  <meta name="msapplication-config" content="/resources/favicons/browserconfig.xml">
+  <meta name="theme-color" content="#ffffff">
+</head>
+<body>
+  
+  <header>
+    <div class="container flex-box head-box">
+      <h1><a href="#"><img src="resources/images/logo.jpg" alt="Rmedia"></a></h1>
+      <input type="search" name="search" id="search" placeholder="search...">
+    </div>
+  </header>
+
+  <section class="content">
+
+    <div class="container">
+
+      <div class="content-section flex-box">
+        <section class="left-section">
+          <div class="drop-down">
+            <button class="drop-down-button">Menu</button>
+          </div>
+          <nav class="navigation">
+            <ul id="nav-menues">
+              
+            </ul>
+          </nav>
+        </section>
+        <section class="middle-section">
+          <div class="top-section">
+            <div class="content-area">
+              <h2>Contact</h2>
+            </div>
+          </div>
+          <div class="bottom-section">
+            <form class="form">  
+
+              <!-- Name Field -->
+              <div class="label-box flex-box">
+                <label for="name">Name</label>
+                <p id="name-counter">25</p>
+              </div>
+              <input type="text" name="name" class="input" id="name" maxlength="25" required>
+              <p class="label" id="name-error"></p>
+              
+              <!-- Subject Field -->
+              <div class="label-box flex-box">
+                <label for="Subject">Subject (Not Required)</label>
+                <p id="subject-counter">25</p>
+              </div>         
+              <input type="text" class="input" name="Subject" id="subject" maxlength="25">
+              <p class="label" id="subject-error"></p>
+              
+              <!-- Phone Field -->
+              <div class="label-box flex-box">
+                <label for="Phone">Phone</label>
+              </div>
+              <input type="text" name="Phone" class="input" id="phone" maxlength="10" required>
+              <p class="label" id="phone-error"></p>
+              
```

Comment:
@Thejus5 Could have added +91 as a formatted prefix in the input, so that validation for it need not have been handled.